rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -1558,6 +1558,62 @@ void should_ForceRequestChanges_When_CurrentUserIsNotTheOwner() {
     assertThat(repModel.getState()).isEqualTo(TaskState.READY);
   }
 
+  @Test
+  void should_CompleteTask() {
+    String url =
+        restHelper.toUrl(RestEndpoints.URL_TASKS_ID, "TKI:000000000000000000000000000000000102");
+    HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("user-1-2"));
+
+    // retrieve task from Rest Api
+    ResponseEntity<TaskRepresentationModel> getTaskResponse =
+        TEMPLATE.exchange(url, HttpMethod.GET, auth, TASK_MODEL_TYPE);
+    assertThat(getTaskResponse.getBody()).isNotNull();
+    TaskRepresentationModel repModel = getTaskResponse.getBody();
+    assertThat(repModel.getState()).isEqualTo(TaskState.CLAIMED);
+    assertThat(repModel.getOwner()).isEqualTo("user-1-2");
+
+    // request changes
+    String url2 =
+        restHelper.toUrl(
+            RestEndpoints.URL_TASKS_ID_COMPLETE, "TKI:000000000000000000000000000000000102");
+    ResponseEntity<TaskRepresentationModel> requestedChangesResponse =
+        TEMPLATE.exchange(url2, HttpMethod.POST, auth, TASK_MODEL_TYPE);
+
+    assertThat(requestedChangesResponse.getBody()).isNotNull();
+    assertThat(requestedChangesResponse.getStatusCode()).isEqualTo(HttpStatus.OK);
+    repModel = requestedChangesResponse.getBody();
+    assertThat(repModel.getOwner()).isEqualTo("user-1-2");
+    assertThat(repModel.getState()).isEqualTo(TaskState.COMPLETED);
```

Comment:
please adjust the instance variables + comment to match with the complete operation